doc/libpmemblk.3

```diff
@@ -254,6 +255,21 @@ at a later time using
 .BR pmemblk_open ()
 as described above.
 .PP
+.BI "size_t pmemblk_bsize(PMEMblkpool *" pbp );
+.IP
+The
+.BR pmemblk_bsize ()
+function returns block size of specified block memory pool.
```

Comment:
Just some grammar nits.  How about:
"... returns *the* block size of *the* specified block memory pool."